rfps/rfp-fvm.md

```diff
@@ -0,0 +1,108 @@
+# FVM Tooling & Infrastructure RFP
+
+## Overview
+
+We are seeking proposals for developer and infrastructure tooling to extend the functionality and usability of the forthcoming [Filecoin Virtual Machine](https://fvm.filecoin.io) (FVM).
+
+The Filecoin Virtual Machine (FVM) combines smart contracts with co-location of storage and compute capabilities.
+
+The FVM is a WASM-based polyglot execution environment for IPLD data. It is designed to support native Filecoin actors written in languages that compile to WASM, as well as smart contracts written for foreign runtimes including the Ethereum Virtual Machine (EVM), Secure EcmaScript (SES), and eBPF.
+
+
+You can also consider contributing via the [FVM Bug Bounties](https://immunefi.com/bounty/filecoin/) program.
+
+## Description
+
+We seek to fund the creation of world-class tools that enable future developers to build on the FVM easily and seamlessly.
+
+We are therefore seeking proposals for various developer and infrastructure tooling to extend the functionality and usability of the FVM.
+
+These projects are essential to build the minimum set of tools that we would need to provide in order to have an end-to-end builders process for people who are deploying smart contracts on FVM.
```

Comment:
Rephrase suggestion: 

These projects are essential for providing the minimum set of tools needed in order to have an end-to-end process for builders deploying smart contracts on FVM.